dealers_choice.js

```diff
@@ -11,26 +11,313 @@ app.use(staticMiddleWare);
+          <div class="regular card">
+            <h2><i class="fas fa-tachometer-alt"></i> 
+            <br />
+            Skill Levels</h2>
+            Find climbs from beginner to expert
+            <div class="slidecontainer">
+              <input type="range" min="1" max="4" value="1" class="slider" id="myRange">
+              <ul id="slider-ul">
+                <li id="easy" value="1">Easy</li>
+                <li id="intermediate" value="2">Intermediate</li>
+                <li id="hard" value="3">Hard</li>
+                <li id="extreme" value="4">Extreme</li>
+              </ul>
+            </div>
+            <div class="button" id="difficultyButton">
+              Find ranges by skill level <i class="fas fa-long-arrow-alt-right" style="font-size: 1em;"></i>
+            </div>
+          </div>
+          <div class="card" id="cta">
+            <h2><i class="fas fa-rocket" id="rocket"></i>
+            <br />Find your mountains! <i class="fas fa-long-arrow-alt-right" style="font-size: 1em;"></i></h2>
+          </div>
+          <div class="card" id="random">
+            <a href="/entries/${randomIdx()}">Not sure where to start? <br />Take a random pick!</a>
+          </div>
+        </div>
+        <div class="sub-heading">
+          <h2>Or check out all of the Fourteeners:</h2>
+        </div>
+        <div id="entry-list">
+          ${entries.map( entry => `
+            <div class="entryContainer ${entry.Difficulty} ${entry["Mountain Range"]}" id="entry${entry.ID}" difficulty="${entry.Difficulty}" mountainRange="${entry['Mountain Range']}">
+              <h2>${entry["Mountain Peak"]}</h2>
+              <p>Range: ${entry["Mountain Range"]}</p>
+              <p>Elevation: ${entry.Elevation_ft}</p>
+              <div id="button-${entry.ID}" class="details-container"><a href="/entries/${entry.ID}">See details</a></div>
+            </div>
+          `).join("")}
+        </div>
+    </div>
     </body>
+    <script> 
+      const allCheckBoxes = document.querySelectorAll('input[type=checkbox]');
+      const skillSlider = document.getElementById("myRange");
+      const ctaButton = document.getElementById("cta");
+      const difficultyButton = document.getElementById("difficultyButton");
+      const sliderUl = document.getElementById("slider-ul");
+      const sliderUlChildren = sliderUl.childNodes;
+      const entryList = document.getElementById("entry-list");
+
+      difficultyButton.addEventListener("click", function(event) {
+        let skillValue = skillSlider.value;
+        window.location.href = "/entries/difficulty/"+skillValue;
+      })
+
+      skillSlider.oninput = function() {
+        const easy = document.getElementById("easy");
+        const intermediate = document.getElementById("intermediate");
+        const hard = document.getElementById("hard");
+        const extreme = document.getElementById("extreme");
+
+        const sliderLabels = [easy, intermediate, hard, extreme];
```

Comment:
in your html, you can refactor the buttons like-
```html
const difficulties = ['easy', 'medium', 'hard', 'extreme'];
<ul id="skill_buttons">
  ${difficulties.map((difficulty, index) => `<li value=${index}>${button}</li>`}
</ul>
```

and in this code you can have something like 
```js
const sliderLabels = document.querySelector('#skill_buttons').childNodes;
```

to limit repetition